### PR TITLE
Homematic: Replace nanohttpd by Jetty Server

### PR DESCRIPTION
**Pull request by MHerbst**
The nanohttpd server has got some problems on Windows OS. It was replaced by the Jetty server. 

Solves issue #1036 

Signed-off-by: Martin Herbst <mail@mherbst.de> (github: mherbst)

### CODE REVIEW

**Comment by kaikreuzer** [comment]
There is a conflict, could you try to rebase your branch on the current master?

File: addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/server/XmlRpcServer.java [hunk 1]
```diff
@@ -58,60 +64,68 @@ public XmlRpcServer(RpcEventListener listener, HomematicConfig config) {
...
                 }
-                return newFixedLengthResponse(returnValue);
-            } catch (Exception ex) {
+                respWriter.println(returnValue);
+            } catch (SAXException | ParserConfigurationException ex) {
                 logger.error(ex.getMessage(), ex);
-                return newFixedLengthResponse(XML_EMPTY_STRING);
+                respWriter.println(XML_EMPTY_STRING);
             }
+            response.setStatus(HttpServletResponse.SC_OK);
```
Comment (gerrieg) [inline addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/server/XmlRpcServer.java]: double: response.setStatus(HttpServletResponse.SC_OK);

**Comment by MHerbst** [comment]
Have removed the duplicate line

**Comment by kaikreuzer** [comment]
No need to close the PR though...?

**Comment by MHerbst** [comment]
Remove the line and accidentially closed the PR. Wrong button pressed.

**Pull request merged**